src/utilities/utilities.jl

```diff
@@ -0,0 +1,2 @@
+private_compat_helper(; env::AbstractDict=ENV) = haskey(env, "PRIVATE_COMPAT_HELPER") && env["PRIVATE_COMPAT_HELPER"] != "false"
```

Comment:
These env queries use the string `PRIVATE_COMPAT_HELPER` but in `CompatHelper.jl` you create a const for `PRIVATE_COMPAT_HELPER` that points to `"COMPATHELPER_PRIV"` should this be using the const instead of the string here?

Reply:
Yes, must have missed this going over it in the last few days. Revising now.